EnglishFilter/sections/foreign.txt

```diff
@@ -3779,6 +3779,9 @@ krucil.net##img[width="900"][height="50"]
 !------------------------------ Hebrew -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/116912
+||st1.bhol.co.il/js/floatingBannerNewSite.js
```

Comment:
It seems we can just block these domains(I didn't find anything broken)
```
||ban.bhol.co.il^
||ext.bhol.co.il^
||st1.bhol.co.il^
```

ad leftover
`bhol.co.il##div[id^="interstitial-caption-"]`